docs/technical_docs/Inji_Verify_API_Overview.md

```
@@ -263,18 +263,18 @@ Wallet submits the Verifiable Presentation. Accepts `application/x-www-form-urle
 | `error` | One of | Wallet error code (mutually exclusive with `vp_token`) |
 | `error_description` | No | Human-readable error detail |
 
-**Response — cross-device / mobile wallet** (`responseCodeValidationRequired=false`)
+**Response — cross-device** (`responseCodeValidationRequired=false`)
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)